lib/protocol/networks.js

```diff
@@ -775,8 +775,8 @@ regtest.block = {
   bip65hash: null,
   bip66height: 1251,
   bip66hash: null,
-  pruneAfterHeight: 1000,
-  keepBlocks: 10000,
+  pruneAfterHeight: 500,
```

Comment:
Minor backward incompat note.

Reply:
Oh yeah, add a note to `/CHANGELOG.md` ?